### ISSUE TITLE
Title attribute or props spread on img element

### ISSUE THREAD

**Issue by Schnueggel** [wiredmax/react-flags#6]
HI, 

how about adding the title attribute or spreading the rest of props into the img

**Comment by wiredmax** [comment]
Good idea, I'll put some love in the small project over the next days and integrate this.
